neuralmonkey/train.py

```diff
@@ -51,14 +52,19 @@ def create_config() -> Configuration:
 
 # pylint: disable=too-many-statements
 def main() -> None:
-    if len(sys.argv) != 2:
-        print("Usage: train.py <ini_file>")
-        exit(1)
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument('config', metavar='INI-FILE',
```

Comment:
co přesně tohle znamená? je to option nebo pojmenovanej parametr? Mění to nějak způsob spouštění pokud nechci měnit žádný konfigurace?